site/content/docs/manifest/rd-web-service.en.md

```diff
@@ -222,6 +222,11 @@ Key-value pairs representing AWS tags that are passed down to your AWS App Runne
 
 <div class="separator"></div>
 
+<a id="count" href="#count" class="field">`count`</a> <span class="type">String</span>
+Specify the name of an existing autoscaling configuration to your request-driven web services.
```

Comment:
I don't think we need this because it's already on the RDWS manifest page. Should we give an example?
```suggestion
Specify the name of an existing autoscaling configuration.
```

Reply:
yeah just added an example